suite/tests/client-interface/cleancall-opt-shared.h

```diff
@@ -718,3 +724,48 @@ codegen_inscount(void *dc)
     codegen_epilogue(dc, ilist);
     return ilist;
 }
+
+/*
+X86
+  bbcount:
+      push REG_XBP
+      mov REG_XBP, REG_XSP
+      inc [global_count]
+      leave
+      ret
+
+AArch64
+   bbcount:
+     x0 and x1 are used as scratch registers
+     mov &global_count to x0 using a series of movz and movk instructions
+     ldr x1, [x0]
+     add x1, x1, #1
+     str [x0], x1
+
+*/
+static instrlist_t *
+codegen_bbcount(void *dc)
+{
+    instrlist_t *ilist = instrlist_create(dc);
+#ifndef X86
+  reg_t reg1 = DR_REG_X0;
+  reg_t reg2 = DR_REG_X1;
+#endif
+
+    codegen_prologue(dc, ilist);
+#ifdef X86
+    APP(ilist, INSTR_CREATE_inc(dc, OPND_CREATE_ABSMEM(&global_count, OPSZ_PTR)));
+#else
+    instrlist_insert_mov_immed_ptrsz(dc, (ptr_int_t)&global_count,
```

Comment:
Yep, it is supposed to be code that is the target of a clean call, i.e. unmanaged and genereated by the compiler.